skins/cat17/templates/page_layouts/use_of_resources.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends 'page_layouts/base.html.twig' %}
+{% set page_works_without_js = false %}
+
+{% block main_content %}
+    <div id="resources" data-content="{$ use_of_resources_content $}"></div>
```

Comment:
Maybe add a `<noscript>` section with links to another place where this info is stored?